ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py

```diff
@@ -3859,3 +3859,33 @@ def test_tensorflow_Zeta(
         x=x[0],
         q=x[1],
     )
+
+
+# Where
+@handle_frontend_test(
+    fn_tree="tensorflow.raw_ops.Where",
+    dtype_and_input=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("integer"),
```

Comment:
The original functions seems to support all dtypes, ideally we should use the default value for `helpers.get_dtypes` which is `valid`

Reply:
okay good point